src/client/components/UnreadMessages/CountUnreadMessages.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import './Unread.style.css';
+import { useParams } from 'react-router-dom';
+
+function CountUnreadMessages() {
+  const { userId, channelId } = useParams();
+
+  const [unreadMessages, setUnreadMessages] = useState([]);
+
+  useEffect(() => {
+    let mounted = true;
+    if (mounted) {
+      const unReadedMessages = async () => {
```

Comment:
`unread` is a word, do not capitalize `R`.
Also, the method name should be `fetchUnreadMessages` instead of this one. Verbs are used for functions, nouns for components.